src/OSPSuite.R/Services/DataImporterTask.cs

```diff
@@ -8,6 +8,10 @@
 using System.Xml.Linq;
 using ImporterConfiguration = OSPSuite.Core.Import.ImporterConfiguration;
 using IContainer = OSPSuite.Utility.Container.IContainer;
+using OSPSuite.Core.Import;
```

Comment:
ImporterTaks and ConfigurationTaks have been combined as they used some shared fields.